WeScan/Review/ReviewViewController.swift

```diff
@@ -103,14 +103,19 @@ final class ReviewViewController: UIViewController {
     }
     
     private func setupConstraints() {
-        let imageViewConstraints = [
-            imageView.topAnchor.constraint(equalTo: view.topAnchor),
-            imageView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
-            view.bottomAnchor.constraint(equalTo: imageView.bottomAnchor),
-            view.leadingAnchor.constraint(equalTo: imageView.leadingAnchor)
-        ]
+        imageView.translatesAutoresizingMaskIntoConstraints = false
         
-        NSLayoutConstraint.activate(imageViewConstraints)
+        if #available(iOS 11.0, *) {
+            view.safeAreaLayoutGuide.leadingAnchor.constraint(equalTo: imageView.safeAreaLayoutGuide.leadingAnchor).isActive = true
+            view.safeAreaLayoutGuide.topAnchor.constraint(equalTo: imageView.safeAreaLayoutGuide.topAnchor).isActive = true
+            view.safeAreaLayoutGuide.trailingAnchor.constraint(equalTo: imageView.safeAreaLayoutGuide.trailingAnchor).isActive = true
+            view.safeAreaLayoutGuide.bottomAnchor.constraint(equalTo: imageView.safeAreaLayoutGuide.bottomAnchor).isActive = true
+        } else {
+            view.leadingAnchor.constraint(equalTo: imageView.leadingAnchor).isActive = true
+            view.topAnchor.constraint(equalTo: imageView.topAnchor).isActive = true
+            view.trailingAnchor.constraint(equalTo: imageView.trailingAnchor).isActive = true
+            view.bottomAnchor.constraint(equalTo: imageView.bottomAnchor).isActive = true
```

Comment:
Can we use `NSLayoutConstraint.activate` instead in both cases? That's a bit more performant!

Reply:
Done ✅